strider/util.py

```diff
@@ -243,3 +243,7 @@ def remove_null_values(obj):
         ]
     else:
         return obj
+
+
+def last(alist, n):
+    return alist[:-n] if n else alist[:]
```

Comment:
This is confusing me. You're not getting the last `n`, you're getting the first `len(alist) - n`. Is that the intent?